src/day-1-toy/func.py

```diff
@@ -1,6 +1,16 @@
 # Write a function is_even that will return true if the passed in number is even.
 
+def is_even(x):
+  return x%2==0
+
 # Read a number from the keyboard
 num = input("Enter a number: ")
 
```

Comment:
As mentioned earlier `input()` returns a string after receiving user input. You might need to do some type handling here as well.